### PR TITLE
Add keypress event callbacks to EditableField

### PR DESCRIPTION
**Pull request by knicklabs**
We need a way to be able to add event listeners to EditableField inputs on key events. This update adds `onInputKeyDown`, `onInputKeyPress`, and `onInputKeyUp` event listeners.

### CODE REVIEW

**Comment by knicklabs** [comment]
Not sure why this build is failing. Will look into it.

**Comment by knicklabs** [comment]
Build failure is all sorted out.

File: src/components/EditableField/EditableField.tsx [hunk 1]
```diff
@@ -252,7 +256,7 @@ export class EditableField extends React.Component<
             value: fieldValue,
             data: {
               cause: 'BLUR',
-              operation: 'UPDATE',
+              operation: OPERATION.UPDATE,
```
Comment (plbabin) [inline src/components/EditableField/EditableField.tsx]: yay!!

File: src/components/EditableField/EditableField.Input.tsx [hunk 2]
```diff
@@ -175,15 +176,19 @@ export class EditableFieldInput extends React.Component<InputProps> {
   /* istanbul ignore next */
   handleKeyDown = event => {
     const isEnter = event.key === key.ENTER
-    const isEscape = event.key === key.ESCAPE
     const isDropdownTrigger = event.target.classList.contains(
       OTHERCOMPONENTS_CLASSNAMES.dropdownTrigger
     )
```
Comment (plbabin) [inline src/components/EditableField/EditableField.Input.tsx]: I am curious on why the Escape part was removed?

Comment (knicklabs) [reply src/components/EditableField/EditableField.Input.tsx]: Before this input was only allowing `enter` and `escape` key events to bubble up to the parent. In the case of enter, it was only bubbling up when no in a dropdown trigger. With the exception of the dropdown case, we now want all key events to bubble up.

**Pull request merged**